packages/playwright/tests/axe-playwright.spec.ts

```diff
@@ -459,48 +459,38 @@ describe('@axe-core/playwright', () => {
 
     it('with include using an array of strings', async () => {
       await page.goto(`${addr}/context.html`);
-      const expected = ['.selector-one', '.selector-two', '.selector-three'];
-
-      const axeSource = `
-      window.axe = {
-        configure(){},
-          run({ include }){
-            return Promise.resolve({ include })
-          }
-      }
-    `;
-      const results = new AxeBuilder({ page, axeSource: axeSource }).include([
-        '.selector-one',
-        '.selector-two',
-        '.selector-three'
-      ]);
+      const results = await new AxeBuilder({ page })
+        .include(['.include', '.include2'])
+        .analyze();
 
-      const { include: actual } = (await results.analyze()) as any;
+      const flattenTarget = flatPassesTargets(results);
 
-      assert.deepEqual(actual[0], expected);
```

Comment:
We would need to keep this test, it checks that the selectors are getting passed to axe-core